src/refinement/amr_criteria.cpp

```diff
@@ -55,7 +55,12 @@ AMRFirstDerivative::AMRFirstDerivative(ParameterInput *pin, std::string &block_n
 }
 
 AmrTag AMRFirstDerivative::operator()(std::shared_ptr<MeshBlockData<Real>> &rc) {
-  ParArrayND<Real> q = rc->Get(field).data;
+  ParArrayND<Real> q;
+  try {
+    q = rc->Get(field).data;
+  } catch (std::invalid_argument const &) {
+    return AmrTag::derefine;
```

Comment:
Not sure if this is what we want. We probably want `AmrTag::same`. @Yurlungur thoughts?

Reply:
@jdolence wrote this so he may have more insight. But I am inclined to agree. I think this should be `AmrTag::same`.